vulcan/lib/server.rb

```diff
@@ -17,6 +17,7 @@ class Server < Etna::Server
 
     get 'api/workflows', action: 'workflows#fetch', as: :workflows_view, auth: { user: { active?: true } }
```

Comment:
Note, I basically expect to throw all these server-side routes out, after we hook into Archimedes.

Reply:
My current understanding is Vulcan will hold onto workflow documents and orchestrate execution while Archimedes just receives and executes cells + stores data, so some of this might remain.

Reply:
That'd be fine, though probably need some clarification since I thought the workflow CWL docs would be held elsewhere. Perhaps that's the "Vulcan - Archimedes Bridge" card in the Kanban board? Just want to make sure we know which component has responsibility for what.

Reply:
Hey @coleshaw 

My goal in terms of responsibility is for `ruby` to hold most of the logic around workflow definitions, session stage, storage management, and scheduling.  archimedes is intended to be `stupid as balls`.  It has one job -- run some python code in a sandbox with with some input / output specifications.  So actually, vulcan will take some responsibilities, including storage of CWL documents.  We (the backend team) will be building that out.  I need to proliferate more cards on this point.

Reply:
Then the Vulcan APIs should definitely be re-worked from their current implementation.